config/environments/test.rb

```diff
@@ -43,8 +45,17 @@
   # Print deprecation notices to the stderr.
   config.active_support.deprecation = :stderr
 
+  # Raise exceptions for disallowed deprecations.
+  config.active_support.disallowed_deprecation = :raise
+
+  # Tell Active Support which deprecation messages to disallow.
+  config.active_support.disallowed_deprecation_warnings = []
+
   # Raises error for missing translations.
-  # config.action_view.raise_on_missing_translations = true
+  # config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
```

Comment:
what does this... do... I realize it isn't enabled but I'm now curious

Reply:
It adds comments to the HTML in test mode, like this!

```html
<!-- BEGIN app/views/layouts/partials/_header.html.erb -->
  <header></header>
<!-- END app/views/layouts/partials/_header.html.erb -->
```
Just to help understand what's going wrong - I guess it's useful for view specs, not that we really have those =)